p/themes/Swage/swage.rtl.css

```diff
@@ -63,7 +63,7 @@ a.btn:hover {
 a#btn-subscription {
   width: 76%;
 }
-a#btn-importExport {
```

Comment:
Ah, it needs to be changed in the `*.scss` files as well (Ansum, Mapco, Swage themes)

Reply:
done